docs/README.md

```diff
@@ -11,9 +11,9 @@ The model data itself is pulled from one or more external [storage instances](pr
 ModelMesh Serving makes use of two core Kubernetes Custom Resource types:
 
 - `ServingRuntime` - Templates for Pods that can serve one or more particular model formats. There are three "built in" runtimes that cover the out-of-the-box model types (Triton, MLServer and OpenVINO Model Server OVMS), [custom runtimes](runtimes/) can be defined by creating additional ones.
-- [`Predictor`](predictors/) - This represents a logical endpoint for serving predictions using a particular model. The Predictor spec specifies the model type, the storage in which it resides and the path to the model within that storage. The corresponding endpoint is "stable" and will seamlessly transition between different model versions or types when the spec is updated.
+- [`InferenceService`](predictors/) - This is the main interface KServe uses for managing models on Kubernetes. ModelMesh Serving can be used for deploying `InferenceService` predictors which represent a logical endpoint for serving predictions using a particular model. The `InferenceService` predictor spec specifies the model format, the storage location in which the model resides, and other optional configuration. The corresponding endpoint is "stable" and will seamlessly transition between different model versions or types when the spec is updated. Note that many features like transformers, explainers, and canary rollouts do not currently apply or fully work using InferenceServices with `deploymentMode` set to `ModelMesh`. And `PodSpec` fields that are set in the `InferenceService` predictor spec will be ignored.
```

Comment:
I am not sure the "InferenceService link" should point to the ModelMesh predictors docs. I understand ModelMesh implements only the predictor component in the KServe InferenceService spec. But it feels a little strange to follow the link and see nothing about InferenceService. I would suggest either remove the link or have it point to KServe InferenceService.